lib/python/ensembl/brc4/runnable/amend_genome_metadata.py

```diff
@@ -0,0 +1,187 @@
+#!env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from pathlib import Path
+import re
+from typing import Any, Dict, List, Tuple
+import eHive
+import gzip
+import csv
+
+from Bio import SeqIO
+from ensembl.brc4.runnable.utils import Utils
```

Comment:
```suggestion
import csv
import gzip
import os
from pathlib import Path
import re
from typing import Any, Dict, List, Tuple

from Bio import SeqIO
import eHive

from ensembl.brc4.runnable.utils import Utils
```
PEP8 suggests three blocks: core libraries, 3rd-party libraries and your own library imports, all in alphabetical order (to ease readability).